ProductsWebApi/ProductsWebApi.Tests/ProductsControllerTest.cs

```diff
@@ -0,0 +1,230 @@
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using ProductsWebApi.Controllers;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ProductsWebApi.Tests
+{
+    public class ProductControllerTest
+    {
+        private readonly Mock<IProductsService> _mockProductsService;
+        private ProductsController _productsController;
+        private readonly Fixture _fixture;
+        private readonly Mock<ILogger<ProductsController>> _logger;
+
+        public ProductControllerTest()
+        {
+            _mockProductsService = new Mock<IProductsService>();
+            _logger = new Mock<ILogger<ProductsController>>();
+            _productsController = new ProductsController(_mockProductsService.Object, _logger.Object);
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task GetProducts_ReturnsOkObjectResult()
+        {
+            //Arrange
+            var expectedResult = _fixture.Build<Product>().CreateMany().ToList();
+
+            _mockProductsService.Setup(x => x.GetProducts()).ReturnsAsync(expectedResult);
+
+            //Act
+            var actualResult = Assert.IsType<OkObjectResult>(await _productsController.GetProducts());
+
+            //Assert
+            Assert.Equal(StatusCodes.Status200OK, actualResult.StatusCode);
+
+            Assert.IsType<List<Product>>(actualResult.Value);
+
+            Assert.NotNull(actualResult.Value);
+
```

Comment:
Why new line between each LOC 🤔 
I meant new line only between arrange act and assert as a sections

//arrage
..
..

//act
...
....
...

//assert
..
..
...
..